x-pack/plugins/security_solution/public/management/pages/policy/view/policy_hooks.ts

```diff
@@ -101,3 +101,8 @@ export function usePolicyDetailsArtifactsNavigateCallback(listId: string) {
     [getPath, history]
   );
 }
+
+export const useIsPolicySettingsBarVisible = () => {
+  const policySettingsRegex = /administration\/policy\/[A-Za-z0-9-]+/;
```

Comment:
Is it possible to do this without a regex?  I'm thinking you could do this by checking a simple string comparison to see if the URL contains `/administration/policy/` and `settings`.

My reasoning is that the bar shows up in URLs like this: `app/security/administration/policy/33ec2ff3-4275-4021-8e64-0633abda8663/settings`

So we could do a comparison like:

`return window.location.pathname.includes(`/administration/policy`) && window.location.pathname.includes(`/settings`)`

It's messier for sure, but remove the regex comparison.

In any case, this check does seem OK given the test from here (linear performance): https://devina.io/redos-checker

<img width="819" alt="image" src="https://user-images.githubusercontent.com/56395104/167218420-9dcb81d0-c254-4b3d-b8a3-73b8dcc6318f.png">

However, since this is checked often, I wonder if we could simplify it.